src/Http/Cookie/CookieInterface.php

```diff
@@ -114,7 +114,7 @@ public function getExpiry();
     /**
      * Get the timestamp from the expiration time
      *
-     * @return int|null The expiry time as a string timestamp.
+     * @return int|null The expiry time as an integer.
```

Comment:
How can this be null? I understand that 0 exists as unlimited, but what is null?

Reply:
It's probably null because setcookie() allows the $expires argument to be null.